src/components/Chart/Maps.layout.js

```diff
@@ -52,7 +53,7 @@ export default (props, geoJson) => {
   const numberFormat = getFormat(props.numberFormat, t)
   let domain
   let colorScale
-  let colorAccessor = d => d.value
+  let colorAccessor = d => d[color] || 0
```

Comment:
`choropleth` are still broken, most likely because of this.

```suggestion
  let colorAccessor = choropleth ? d => d.value : d => d[color] || 0
```